components/CommentCount.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+class CommentCount extends React.Component {
+  formatString (count) {
+    if (count == undefined) { count = 0 }
```

Comment:
My answer ties into both these comments - I'm not sure, cos with the stories it feels like a "not upvoted" story doesn't care whether it's got comments or not. Maybe the stories need to be more all-encompassing? In any case, that's the situation when `commentCount` would be undefined (I ran into it).